### ISSUE TITLE
Expose @volar/server/out/index.js as executable in $NPM_PREFIX/bin 

### ISSUE THREAD

**Issue by 3nuc** [johnsoncodehk/volar#458]
I was trying to add Volar as a language server for nvim-lspconfig and I got hung up for a second on the part where you need to tell neovim which executable to launch. I later figured it out and this issue is a nitpick, but still it removes a tiny hurdle. 

After you run `npm i -g @volar/server` there's no binary added to the NPM bin/ folder, so you can't easily launch it. On the other hand with Vetur's language server, after global install you have the `vuels` command.

The Vim LSP integrations for Volar work around this by creating wrapper scripts. They just execute the server/out/index.js with Node and passthrough any arguments.

Examples:
https://github.com/ouuan/AUR-packages/blob/master/volar-server/wrapper.sh
https://github.com/mattn/vim-lsp-settings/blob/1fdda7d3493f086b5e28446f8daa6a9d6dc1325c/installer/install-volar.sh#L16

I'm not familiar with are the executable files created but Vetur's LS has this: https://github.com/vuejs/vetur/blob/master/server/package.json#L9

Basically, the intended effect is that you can run `volar-server` from the command line after running `npm i -g @volar/server`. Which is not possible now.

**Comment by johnsoncodehk** [comment]
Thank you for your suggestion. What is the expected cli command? `volar-server` / `@volar/server` / `volar`?

@ouuan, @tsukkee do you agree add this?

**Comment by 3nuc** [comment]
It doesn't really matter as this is a transparent change (ie. doesn't break existing LSP integrations for third-parties). [I'm not sure if `@volar/server` will work though](https://askubuntu.com/a/719922/963658) - even `npm i -g @vue/cli` expoes a `vue` (eg `vue create`) executable.

Whatever is more aesthetically pleasing to you all :P May be worth considering `volar-vue-ls` so nobody gets the idea that "hmm what's this new 'Volar' language", since languageservers are named like `[languagename]-language-server`.

**Comment by johnsoncodehk** [comment]
I decide use `vue-server` command for now, could you test `@volar/server@0.27.14-alpha.1`?

**Comment by tsukkee** [comment]
Sorry for my late reply.

https://github.com/johnsoncodehk/volar/commit/7734d1e1993b05aa62b35012329e8f5ab27f2a80 looks fine.
It makes the volar setting on vim-lsp-settings simpler.

**Comment by ouuan** [comment]
> I decide use `vue-server` command for now, could you test `@volar/server@0.27.14-alpha.1`?

I think it's confusing with vetur, whose command is `vls` (Vue Language Server).

**Comment by johnsoncodehk** [comment]
> I think it's confusing with vetur, whose command is `vls` (Vue Language Server).

It's no problem I think. 🤔

**Comment by johnsoncodehk** [comment]
Change command `vue-server` to `volar-server`, because I want to rename `@volar/server` on version 1.0 or earlier, use `volar-server` for `@volar/server` is easier to replace it. And we have more time to decide the official command.

**Comment by tsukkee** [comment]
>  It makes the volar setting on vim-lsp-settings simpler. I'll try @volar/server@0.27.14-alpha.1 later.

I confirmed it works fine with vim-lsp and sent PR to vim-lsp-settings with it ( https://github.com/mattn/vim-lsp-settings/pull/458 ).

**Comment by johnsoncodehk** [comment]
Please note #876. :)
